CMakeLists.txt

```diff
@@ -59,7 +59,7 @@ set(CMAKE_MODULE_PATH ${CMAKE_MODULE_PATH} "${CMAKE_SOURCE_DIR}/cmake/Modules/")
 include(${CMAKE_SOURCE_DIR}/cmake/Modules/misc.cmake)
 
 # This is done here so that relative search paths are more reasnable
-find_package(Irrlicht)
+find_package(Irrlicht REQUIRED)
```

Comment:
Irrlicht is required to build the server.  It at least depends on types (`v3s16`, `v3f`, `s8`, `u64`...), although it may depend on more significant parts.